deploy/sre-prometheus/100-cluster-upgrade.PrometheusRule.yaml

```diff
@@ -13,8 +13,7 @@ spec:
     - alert: ClusterUpgradingSRE
       expr: cluster_version{type="updating"} > 0
       labels:
-        severity: warning
+        severity: info
         namespace: openshift-monitoring
       annotations:
-        from_version: "$labels.from_version"
-        to_version: "$labels.version"
```

Comment:
Why remove these annotations?  I assume they're available in PD and we could use it to query telemeter data, but maybe they should be labels for that?

Reply:
The labels already show up in the PD alert, but the annotations don't render. They literally show up in PD as:
```
Annotations:
 - from_version = $labels.from_version
 - to_version = $labels.version
```